src/test/groovy/generated/AbstractGeneratedAstTestCase.groovy

```diff
@@ -6,12 +6,12 @@ import groovy.transform.Generated
 import java.lang.reflect.Constructor
 import java.lang.reflect.Method
 
-/**
- * @author Dmitry Vyazelenko
- * @author Andres Almiray
- */
 @CompileStatic
 abstract class AbstractGeneratedAstTestCase {
+    protected final Class<?> parseClass(String str) {
+        new GroovyClassLoader().parseClass(str.stripMargin())
```

Comment:
That's nicer that it is hidden but you can simply remove it and the'|' characters if you want.

Reply:
Right. We took inspiration on another AST test that also uses '|' and stripMargin, thought it would be OK to keep it consistent.

Reply:
The existing `stripMargin` examples are possibly older tests that we haven't tidied up but for me this falls into "beauty is in the eye of the beholder" and I think we can embrace multiple forms of beauty. :-)

Reply:
Fixed. Also re-instantiated skipped tests as GROOVY-9161/9162/9163 have been fixed in master

Reply:
LGTM